### PR TITLE
calibre312 script - install latest Calibre 3.x on Raspbian

### PR DESCRIPTION
**Pull request by jvonau**

### CODE REVIEW

**Comment by holta** [comment]
Builds off #530 for Raspbian, to install Calibre 3.x (e.g. 3.12 for now...)

File: roles/calibre/tasks/main.yml [hunk 1]
```diff
@@ -1,62 +1,68 @@
...
+    path: "{{ calibre_dbpath }}/metadata.db"
+  register: calibre_db
 
-- name: Install Calibre (CentOS)
-  shell: "{{ downloads_dir }}/calibre-installer.py >> /dev/null"
-  args:
-    creates: /usr/bin/calibre-uninstall
-  when: calibre_install and ansible_distribution == 'CentOS'
+- name: Installing Calibre via calibre-installer.py
+  include_tasks: installing.yml
```
Comment (holta) [inline roles/calibre/tasks/main.yml]: Should this be `include_tasks: installer.yml` (or should the corresponding file be `installing.yml` ?)

Comment (jvonau) [reply roles/calibre/tasks/main.yml]: yup

**Comment by holta** [comment]
Ref: #528 #529 #530

**Comment by holta** [comment]
Tested on Raspbian Lite, Raspbian Desktop & Debian 9.1

**Comment by jvonau** [comment]
I had this as clean PR, what the hell is with all the extra commits now?

**Pull request closed without merging**